app/controllers/openid_connect/logout_controller.rb

```diff
@@ -5,7 +5,7 @@ def index
 
       result = @logout_form.submit
 
-      analytics.track_event(Analytics::OPENID_CONNECT_LOGOUT, result.to_h.except(:redirect_uri))
+      analytics.track_event(Analytics::LOGOUT_INITIATED, result.to_h.except(:redirect_uri))
```

Comment:
Do we want to keep track of whether the logout is OIDC or SAML with an event? I.e. instead of creating a `LOGOUT_INITIATED` event, creating at `SAML_LOGOUT` event?

Reply:
One event makes it easier to compare OIDC vs SAML. In Kibana, you would query a single event name, and then you can filter on `sp_initiated: true` vs `sp_initiated: false`. And then to determine the split between OIDC and SAML, you would further filter on `oidc: true` vs `oidc: false`.

Reply:
It's really just one logout event, but with different properties each time. This is similar to how we track multi-factor authentication. We use a single event, but the method of authentication is captured as an event property, and it can be SMS, Voice, TOTP, or personal key.

Reply:
Cool, that makes sense :+1: